apps/ae_socket_connector/lib/socket_connector.ex

```diff
@@ -904,7 +951,8 @@ defmodule SocketConnector do
     "channels.sign.update",
     "channels.sign.initiator_sign",
     "channels.sign.deposit_tx",
-    "channels.sign.withdraw_tx"
+    "channels.sign.withdraw_tx",
+    "channels.sign.force_progress_tx"
```

Comment:
Thinking that FP doesn't contain round. So it should be in the: `# Note, these doesn't contain round...` section

Reply:
This comes from FSM, the FSM prepares the transaction, updates and of course round is also a part of this. Then the user should like, take a look at everything and sign(agree) or disagree(not to sign) it. More info here: https://github.com/aeternity/protocol/blob/cb4dad7a5cba6c902f76c567cb6180cb4016ccf3/node/api/channel_ws_api.md#forcer-receives-a-prepared-force-progress-transaction-to-authenticate

Reply:
Sorry, misunderstood your comment, will be adjusted soon

Reply:
Actually, after doing some research, it appears that the [`round`](https://github.com/aeternity/aeternity/blob/master/apps/aechannel/src/aesc_force_progress_tx.erl#L351) is in the `aesc_force_progress_tx`